CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ Here is a template for new release sections
  - Output of `scalars.xlsx`now also includes `INSTALLED_CAP` and `LCOE_ASSET`(#438)
 - File "constants_output.py" to contain all keys included in "scalars.xlsx" (#453)
 - Installation help for `pygraphviz` on Win10/64bit systems in `troubleshooting.rst` (#379)
+ - Add Plotly-based plots to `F2_autoreport.py` (#439)
```

Comment:
@mahendrark I noticed you did not consistently use the PR number in the changelog entries (I fixed this during the rebase but I wanted to notify you :))